R/getAliases.R

```diff
@@ -0,0 +1,30 @@
+
+# get installed packages, making sure that high priority packages are on top
+ip <- installed.packages()
+
+ord <- order(ip[, "Priority"])
+ip <- ip[ord, ]
+
+ret <- list()
+
+ret <- lapply(rownames(ip), function(row) {
+    libPath <- ip[row, "LibPath"]
+    pkg <- ip[row, "Package"]
+    filename <- file.path(libPath, pkg, "help", "aliases.rds")
+    info <- list(
+        package = pkg,
+        libPath = libPath,
+        aliasFile = filename,
+        aliases = NULL
+    )
+    if (file.exists(filename)) {
+        info[["aliases"]] <- as.list(readRDS(filename))
+    }
+    info
+})
+names(ret) <- rownames(ip)
+
+json <- jsonlite::toJSON(ret, auto_unbox = TRUE)
```

Comment:
This requires `jsonlite`. If user has not installed this package, then the menu will not show and no error message appears either. Do you think we should handle such case and tell user that `jsonlite` is required?

Reply:
I added a `require('jsonlite')` in the beginning to avoid throwing errors.